test/integration/controller/job/job_controller_test.go

```diff
@@ -845,7 +846,7 @@ var _ = ginkgo.Describe("Job controller interacting with scheduler", ginkgo.Orde
 		util.ExpectAdmittedActiveWorkloadsMetric(prodClusterQ, 1)
 	})
 
-	ginkgo.It("Should unsuspend job iff localQueue is in the same namespace", func() {
+	ginkgo.It("Should unsuspend job if localQueue is in the same namespace", func() {
```

Comment:
Is it a drive by fix, or a legitimate change in semantics. If the former, please do a small separate PR.